app/components/group_list_component/view.html.erb

```diff
@@ -21,7 +21,7 @@
       @groups.each do |group|
         body.with_row do |row|
           row.with_cell { govuk_link_to(group.name, group) }
-          row.with_cell(numeric: true) { t('groups.group_list.created_by_unknown') }
+          row.with_cell(numeric: true) { group.creator.present? ? group.creator&.name : t('groups.group_list.created_by_unknown') }
```

Comment:
Not exactly equivalent but I think you could use [presence](https://apidock.com/rails/Object/presence) here to avoid the ternary operator:

```suggestion
          row.with_cell(numeric: true) { group.creator&.name.presence || t('groups.group_list.created_by_unknown') }
```